### PR TITLE
fix(browser): remove old js errors content

### PR DESCRIPTION
**Pull request by ghost**
Jira: NR-201025

### CODE REVIEW

File: src/content/docs/browser/browser-monitoring/browser-pro-features/upload-source-maps-un-minify-js-errors.mdx [hunk 1]
```diff
@@ -17,30 +17,23 @@ import browserReleaseIds from 'images/browser_screenshot-full_release-ids.webp'
...
-2. From the selected JS error group, select the error instance details tab (next to the **Summary** tab).
-3. Error groupings will have one or more frames listed that reference JavaScript files. Select the + icon on the right side of a frame to expand it. Then drag the matching source map file into the frame. **Alternative option:** when the frame is expanded, select **find file** to upload the file manually.
-
-   <Callout variant="tip">
-     An error frame with a gray strip on the left side indicate minified JS; a blue strip indicates it has had a source map applied.
-   </Callout>
+1. Download your source map on your local machine. 
+2. Go to **[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Browser > (select an app) > Errors**, then click on an error group. (Do not select a group labeled **Errors without a stack trace**.)
+3. In the **Stack trace** section, click the **Upload source map**.
+4. In the file finder popup, select your source file. An error frame with a gray strip on the left side indicate minified JS. A blue strip indicates it has had a source map applied.
```
Comment (ally-sassman) [inline src/content/docs/browser/browser-monitoring/browser-pro-features/upload-source-maps-un-minify-js-errors.mdx:36]:
```suggestion
1. Download your source map on your local machine. 
2. Go to **[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Browser > (select an app) > Errors**, then click on an error group. (Do not select a group labeled **Errors without a stack trace**.)
3. In the **Stack trace** section, click the **Upload source map**.
4. In the file finder popup, select your source file. An error frame with a gray strip on the left side indicates minified JS. A blue strip indicates it has had a source map applied.
```

File: src/content/docs/browser/browser-monitoring/browser-pro-features/upload-source-maps-un-minify-js-errors.mdx [hunk 2]
```diff
@@ -17,30 +17,23 @@ import browserReleaseIds from 'images/browser_screenshot-full_release-ids.webp'
...
 2. Publish source maps to browser via the API
 
-This document will explain the first method: how to upload source maps via the UI, along with general [troubleshooting help](#troubleshoot). For instructions on using the API method, see [Push source maps via the API](/docs/push-source-maps-api).
+This document will explain the first method: how to upload source maps via the UI, along with general [troubleshooting help](#troubleshoot). For instructions on using the API method, see [Push source maps via the API](/docs/browser/new-relic-browser/browser-pro-features/upload-source-maps-api/).
 
 ## Import source maps via the New Relic UI [#source-map-ui]
 
-You can drag and drop, or upload, a source map file into the browser UI to associate it with a specific JavaScript file. New Relic will then convert minified stack traces into un-minified traces and source code visible on the [JS errors page](#examine-unminified).
+You can drag and drop, or upload, a source map file into the browser UI to associate it with a specific JavaScript file. New Relic will then convert minified stack traces into un-minified traces and source code visible on the Errors page.
```
Comment (jeff-colucci) [inline src/content/docs/browser/browser-monitoring/browser-pro-features/upload-source-maps-un-minify-js-errors.mdx:31]: The formatting for the first part of this line is a little hard to parse. Maybe just "You can either drag and drop or upload a source map" and ditch the commas?

Also, should the Errors in Errors page be bold? I'm having a brain fog attack so I can't remember.

File: src/content/docs/browser/browser-monitoring/browser-pro-features/upload-source-maps-un-minify-js-errors.mdx [hunk 3]
```diff
@@ -17,30 +17,23 @@ import browserReleaseIds from 'images/browser_screenshot-full_release-ids.webp'
...
-2. From the selected JS error group, select the error instance details tab (next to the **Summary** tab).
-3. Error groupings will have one or more frames listed that reference JavaScript files. Select the + icon on the right side of a frame to expand it. Then drag the matching source map file into the frame. **Alternative option:** when the frame is expanded, select **find file** to upload the file manually.
-
-   <Callout variant="tip">
-     An error frame with a gray strip on the left side indicate minified JS; a blue strip indicates it has had a source map applied.
-   </Callout>
+1. Download your source map on your local machine. 
+2. Go to **[one.newrelic.com > All capabilities](https://one.newrelic.com/all-capabilities) > Browser > (select an app) > Errors**, then click on an error group. (Do not select a group labeled **Errors without a stack trace**.)
+3. In the **Stack trace** section, click the **Upload source map**.
+4. In the file finder popup, select your source file. An error frame with a gray strip on the left side indicates minified JS. A blue strip indicates it has had a source map applied.
```
Comment (jeff-colucci) [inline src/content/docs/browser/browser-monitoring/browser-pro-features/upload-source-maps-un-minify-js-errors.mdx:36]: popup or pop-up?

**Review by jeff-colucci — DISMISSED** [review]
I saw only a couple of small things but I'm approving anyway. If you decide to change stuff, I think you'll be good to go without a second approval.

File: src/content/docs/errors-inbox/browser-tab.mdx [hunk 4]
```diff
@@ -7,7 +7,8 @@ tags:
 metaDescription: New Relic's browser group errors tab shows trends and anomalies that help you compare and troubleshoot error events.
 redirects:
   - /docs/browser/new-relic-browser/browser-pro-features/javascript-errors-page-detect-analyze-errors
-freshnessValidatedDate: never
+ - /docs/browser/new-relic-browser/browser-pro-features/browser-error-profiles-find-error-causes
```
Comment (ally-sassman) [inline src/content/docs/errors-inbox/browser-tab.mdx:10]:
```suggestion
  - /docs/browser/new-relic-browser/browser-pro-features/browser-error-profiles-find-error-causes
```

File: src/content/docs/browser/browser-monitoring/browser-pro-features/upload-source-maps-un-minify-js-errors.mdx (hunk 3 above)
Comment (ally-sassman) [inline src/content/docs/browser/browser-monitoring/browser-pro-features/upload-source-maps-un-minify-js-errors.mdx:36]:
```suggestion
4. In the file finder pop-up, select your source file. An error frame with a gray strip on the left side indicates minified JS. A blue strip indicates it has had a source map applied.
```

File: src/content/docs/errors-inbox/browser-tab.mdx [hunk 5]
```diff
@@ -7,7 +7,8 @@ tags:
 metaDescription: New Relic's browser group errors tab shows trends and anomalies that help you compare and troubleshoot error events.
 redirects:
   - /docs/browser/new-relic-browser/browser-pro-features/javascript-errors-page-detect-analyze-errors
-freshnessValidatedDate: never
+  - /docs/browser/new-relic-browser/browser-pro-features/browser-error-profiles-find-error-causes
+freshnessValidatedDate: 12-27-2023
```
Comment (ally-sassman) [inline src/content/docs/errors-inbox/browser-tab.mdx:11]:
```suggestion
freshnessValidatedDate: 2024-1-2
```

File: src/content/docs/browser/new-relic-browser/browser-pro-features/upload-source-maps-api.mdx [hunk 6]
```diff
@@ -7,7 +7,7 @@ redirects:
 freshnessValidatedDate: never
```
Comment (ally-sassman) [inline src/content/docs/browser/new-relic-browser/browser-pro-features/upload-source-maps-api.mdx:7]:
```suggestion
freshnessValidatedDate: 2024-01-02
```

File: src/content/docs/errors-inbox/browser-tab.mdx [hunk 7]
```diff
@@ -7,7 +7,8 @@ tags:
 metaDescription: New Relic's browser group errors tab shows trends and anomalies that help you compare and troubleshoot error events.
 redirects:
   - /docs/browser/new-relic-browser/browser-pro-features/javascript-errors-page-detect-analyze-errors
-freshnessValidatedDate: never
+  - /docs/browser/new-relic-browser/browser-pro-features/browser-error-profiles-find-error-causes
+freshnessValidatedDate: 2024-1-2
```
Comment (ally-sassman) [inline src/content/docs/errors-inbox/browser-tab.mdx:11]:
```suggestion
freshnessValidatedDate: 2024-01-02
```
